CHANGELOG.md

```diff
@@ -1,8 +1,11 @@
 # Changelog
 
+## Unreleased
+
 - fix: Scope values should not override Event values (#446)
 - feat: Extend User inteface by adding Data, Name and Segment (#483)
 - feat: Make maximum amount of spans configurable (#460)
+- feat: Add ClientOptions.SendDefaultPii #485
```

Comment:
Happy to see the Sentry Go SDK evolving!

One suggestion: initialisms like "PII" are spelled with a consistent case as per Go community conventions https://github.com/golang/go/wiki/CodeReviewComments#initialisms.

The option could be called `ClientOptions.SendDefaultPII`.

Reply:
Thanks, I opened https://github.com/getsentry/sentry-go/pull/490